apps/src/lab2/views/components/PredictSummary.tsx

```diff
@@ -17,18 +21,53 @@ const PredictSummary: React.FunctionComponent = () => {
     `viewAs=${ViewType.Instructor}`
   );
   const summaryUrl = document.location.pathname + SUMMARY_PATH + params;
+  const currentSectionId = useSelector(
+    (state: {teacherSections: {selectedSectionId: number}}) =>
+      state.teacherSections.selectedSectionId
+  );
+  const currentLevelId = useAppSelector(state => getCurrentLevel(state)?.id);
```

Comment:
any reason not to use `state.progress.currentLevelId`?

Reply:
also just a miss on my part, updating!